readme.md

```diff
@@ -1,15 +1,3 @@
-A conversion of the OpenAI OpenAPI to TypeSpec.
+# OpenAPI spec for the OpenAI API
```

Comment:
This is a weird Windows thing, we think, and I can follow up by deleting this file, which I've moved to conversion.md.